spec/controllers/invitations_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe InvitationsController do
+  describe "GET new" do
+    it "sets @invitations to a new invitation" do
+      set_current_user
+      get :new
+      expect(assigns(:invitation)).to be_new_record
+      expect(assigns(:invitation)).to be_instance_of Invitation
+    end
+
+    it_behaves_like "requires sign in" do
+      let(:action) { get :new }
+    end
+  end
+
+  describe "POST create" do
+    it_behaves_like "requires sign in" do
+      let(:action) { post :create }
+    end
+    context "with valid input" do
+      after { ActionMailer::Base.deliveries.clear }
```

Comment:
Since you do this `config.after(:each) { ActionMailer::Base.deliveries.clear }` in `spec_helper.rb` this becomes useless.